src/cli.ts

```diff
@@ -64,6 +64,10 @@ program.command('fix')
     .description('Checks the project for linting issues, and fixes issues wherever possible.')
     .action(fixCommandProcess);
 
+
+program.command('docs')
+    .description('Generates typedoc for the project using the standardized look.')
```

Comment:
Is 'look' the best term here? 'style? 'standard OpenFin template'?

Reply:
Not exactly an openfin template.  Maybe `theme`?

Reply:
I like theme